torch/fx/experimental/symbolic_shapes.py

```diff
@@ -3333,6 +3333,13 @@ def _maybe_evaluate_static(
                     subst[sympy.Le(-e.lhs, -e.rhs)] = sympy.true
                     subst[sympy.Lt(e.lhs, e.rhs)] = sympy.false
                     subst[sympy.Lt(-e.lhs, -e.rhs)] = sympy.false
+                elif isinstance(e, sympy.Lt):
+                    subst[sympy.Le(e.lhs, e.rhs)] = sympy.true
+                    subst[sympy.Le(-e.lhs, -e.rhs)] = sympy.true
```

Comment:
If `a < b`, then it is NOT true that `-a < -b`, indeed, you need to flip the inequality